snuba/datasets/outcomes.py

```diff
@@ -11,11 +11,18 @@
     UUID,
 )
 from snuba.datasets import Dataset
+from snuba.datasets.dataset_schemas import DatasetSchemas
 from snuba.processor import _ensure_valid_date, MessageProcessor, _unicodify
-from snuba.datasets.schema import MergeTreeSchema
+from snuba.datasets.schema import MergeTreeSchema, SummingMergeTreeSchema, MaterializedViewSchema
 from snuba import settings
 
 
+WRITE_SCHEMA_LOCAL_TABLE_NAME = 'outcomes_raw_local'
```

Comment:
I don't think you need to specify SCHEMA here:
just WRITE_LOCAL_TABLE_NAME.
The schema is built on top of this not the other way around.